### DISCUSSION TITLE
Is there any option for 2 or more depth archive file.

### DISCUSSION THREAD

**Discussion by cbskhan** [BurntSushi/ripgrep#2377] [Q&A]
Can search 2-depth or more compressed files?
[3.zip](https://github.com/BurntSushi/ripgrep/files/10314070/3.zip)

**Comment by BurntSushi** [comment]
[From the FAQ](https://github.com/BurntSushi/ripgrep/blob/master/FAQ.md#how-do-i-search-compressed-files):

> ripgrep currently does not search archive formats, so `*.tar.gz files`, for example, are skipped.
